spring-integration-core/src/main/java/org/springframework/integration/aggregator/ExpressionEvaluatingReleaseStrategy.java

```diff
@@ -35,7 +35,7 @@ public ExpressionEvaluatingReleaseStrategy(String expression) {
 	 * be boolean).
 	 */
 	public boolean canRelease(MessageGroup messages) {
-		return ((Boolean) process(messages.getMessages())).booleanValue();
+		return (Boolean) process(messages.getMessages());
 	}
```

Comment:
Perhaps we should call `setExpectedType(Boolean.class)` in the ctor to avoid a CCE here ?